README.md

```diff
@@ -45,10 +45,25 @@ topos --help
 
 Find more about how topos works in the [documentation](https://docs.topos.technology/).
 
+### Topos Docker image
+
+The `TOOLCHAIN_VERSION` needs to be defined through `(stable|nightly-2022-07-20|...)`
```

Comment:
Why not use `RUSTUP_TOOLCHAIN`, the standard? https://rust-lang.github.io/rustup/overrides.html